staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -21,7 +21,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// Rule is a tuple of APIGroups, APIVersion, and Resources.It is recommended
```

Comment:
Why remove this?

Reply:
Good question, not sure how this happened